assets/app/scripts/directives/overviewDeployment.js

```diff
@@ -27,27 +28,39 @@ angular.module('openshiftConsole')
           $scope.desiredReplicas = null;
         });
 
-        // Debounce scaling so multiple clicks within 500 milliseconds only
-        // result in one request.
+        $scope.canScale = function() {
+          if (!$scope.deploymentConfigId || $scope.deploymentConfigMissing) {
+            // Scale as replication controller.
+            return true;
+          }
+
+          if (!$scope.deploymentConfig) {
+            // Wait for the deployment config.
+            return false;
+          }
+
+          // Only allow scaling of the latest deployment.
+          return deploymentIsLatestFilter($scope.rc, $scope.deploymentConfig);
```

Comment:
Thanks, is there a way to tell if a deployment is active?

We show the latest deployment on the overview and any deployments that have status replicas, but with this change, the scaling controls are only enabled for the latest.